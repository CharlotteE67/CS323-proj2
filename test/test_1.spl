int main(){
    float a;
    a = test_func(b);
}