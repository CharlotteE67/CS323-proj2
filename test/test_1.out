Error type 1 at Line 3: undefined variable b.
Error type 2 at Line 3: undefined function.